### ISSUE TITLE
Web Config Error when installation

### ISSUE THREAD

**Issue by johnkalangha** [osTicket/osTicket#4677]
MYSQL : 5.7
server: IIS
php. 5.6
osticket 1.10.4

is there something wrong in webconfig?
![image](https://user-images.githubusercontent.com/30627667/51958862-0b4b8c80-248d-11e9-9c10-1f725d80ce53.png)
Solution for the meantime is I remove the tag <rewrite> just to proceed in the installation but i think it is important in osticket to hide the url php in bar address?

**Comment by ntozier** [comment]
> is there something wrong in webconfig?

I do not think that I have ever used the webconfig that comes with osTicket.  Whats the malformed XML element in the file?  Maybe its because you are using IIS 10.0 and I think that the webconfig was written back for IIS6 and a syntax has changed.

> just to proceed in the installation but i think it is important in osticket to hide the url php in bar address?

I'm not sure what you mean.

**Comment by johnkalangha** [comment]
yes im using iis10.0, i see, thanks sir

**Comment by ntozier** [comment]
Q: Whats the malformed XML element in the file?

**Comment by AdrianRN** [comment]
Hi, actually I have the same problem, and I'm trying to solve this issue, can you help?
I read "Solution for the meantime is I remove the tag" 
But, I don't know what's the tag.

**Comment by ntozier** [comment]
@AdrianRN This is from 2019 for a version of osTicket that is end of life (1.10.x) and is closed issue.  I do not know that they will respond with clarification of what they did to fix their issue.  

Regardless, I think that you might be better off posting your question over on the support forms. https://forum.osticket.com/
